webapp/channels/src/components/channel_banner/channel_banner.tsx

```diff
@@ -29,7 +31,10 @@ type Props = {
 
 export default function ChannelBanner({channelId}: Props) {
     const channelBannerInfo = useSelector((state: GlobalState) => getChannelBanner(state, channelId));
-    const showChannelBanner = useSelector((state: GlobalState) => selectShowChannelBanner(state, channelId));
```

Comment:
now that the channel banner has this logic here, the responsibility for license validation have been shifted from the selector to the consuming component, but the tests for the license validation didn't follow. The license validation logic is now untested neither in the selector nor in the component.

Reply:
either way, the license validation should be done in both sides, the selector and the component

Reply:
I meant, the license validation tests coverage ^^